en/flight_modes_fw/mission.md

```diff
@@ -0,0 +1,372 @@
+
+The vehicle will follow a smooth rounded curve towards the next waypoint (if one is defined) defined by the acceptance radius ([NAV_ACC_RAD](../advanced_config/parameter_reference.md#NAV_ACC_RAD)).
+The diagram below shows the sorts of paths that you might expect.
+
+![acc-rad](../../assets/flying/acceptance_radius_mission.png)
+
+Vehicles switch to the next waypoint as soon as they enter the acceptance radius:
+
+- For MC this radius is defined by [NAV_ACC_RAD](../advanced_config/parameter_reference.md#NAV_ACC_RAD).
+- For FW the acceptance radius is defined by the "L1 distance".
+  - The L1 distance is computed from two parameters: [NPFG_DAMPING](../advanced_config/parameter_reference.md#NPFG_DAMPING) and [NPFG_PERIOD](../advanced_config/parameter_reference.md#NPFG_PERIOD), and the current ground speed.
+  - By default, it's about 70 meters.
+  - The equation is:
+    $$L_{1_{distance}}=\frac{1}{\pi}L_{1_{damping}}L_{1_{period}}\left \| \vec{v}_{ {xy}_{ground} } \right \|$$
+
+## Mission Takeoff
+
+Starting flights with mission takeoff (and landing using a mission landing) is the recommended way of operating a plane autonomously.
+
+Both [runway takeoff](../flight_modes_fw/takeoff.md#runway-takeoff) and [hand-launched takeoff](../flight_modes_fw/takeoff.md#catapult-hand-launch) are supported — for configuration information see [Takeoff mode (FW)](../flight_modes_fw/takeoff.md).
+
+The takeoff behaviour is defined in a Takeoff mission item, which corresponds to the [MAV_CMD_NAV_TAKEOFF](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_TAKEOFF) MAVLink command.
+During mission execution the vehicle will takeoff towards this waypoint, and climb until the specified altitude is reached.
+The mission item is then accepted, and the mission will start executing the next item.
+
+More specifically, the takeoff mission item defines the takeoff course and clearance altitude.
+The course is the line between the vehicle starting point and the horizontal position defined in the mission item, while the clearance altitude is the mission item altitude.
+
+For a runway takeoff, the `Takeoff` mission item will cause the vehicle to arm, throttle up the motors and take off.
+When hand-launching the vehicle will arm, but only throttle up when the vehicle is thrown (the acceleration trigger is detected).
+In both cases, the vehicle should be placed (or launched) facing towards the takeoff waypoint when the mission is started.
+If possible, always make the vehicle takeoff into the wind.
+
+:::note
+A fixed-wing mission requires a `Takeoff` mission item to takeoff; if however the vehicle is already flying when the mission is started the takeoff item will be treated as a normal waypoint.
+:::
+
+For more infomormation about takeoff behaviour and configuration see [Takeoff mode (FW)](../flight_modes_fw/takeoff.md).
+
+## Mission Landing
```

Comment:
Can an MC follow a mission landing?

Reply:
I'm actually not sure but as far as I'm concerned I'd not document it since I wouldn't know who uses/tests this.

Reply:
Yes, there is no difference between vehicle types afaik there.